src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -833,6 +837,11 @@ void UpdateDimensionProperties()
 
 			TrimOverscroll(Orientation.Vertical);
 			TrimOverscroll(Orientation.Horizontal);
+
+			if (new Size(ExtentWidth, ExtentWidth) is { } newSize && oldSize != newSize)
```

Comment:
While valid, I'd tend to find this less verbose:

```suggestion
            Size newSize = (ExtentWidth, ExtentWidth);
			if (oldSize != newSize)
```

Out of curiosity I tried to see what the compiler does, and it is smart enough to create just a variable initialization:

```
public class C
{
    public void M()
    {
        Size size = new Size(0, 0);
        Size size2 = new Size(0, 0);
        if (!(size != size2))
        {
        }
    }
}
```

Reply:
sadly this wont compile:
> error CS0029: Cannot implicitly convert type '(double, double)' to 'Windows.Foundation.Size'

going back to: var this = new That(Stuffs...)